src/Core/Datatypes/ColorMap.h

```diff
@@ -118,6 +118,53 @@ namespace Datatypes {
   private:
     StandardColorMapFactory() = delete;
   };
+  
+  // colormap helper for ospray transfer function
+  class ColorMap_OSP_helper
```

Comment:
Move all implementation for this class into ColorMap.cc. All the initialization (which will change a lot as more colormaps are added) should not be defined in the header.